quickstarts/microsoft.web/app-service-managed-certificate-apex/azuredeploy.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "SubjectName": {
+            "type": "string",
+            "metadata": {
+                "description": "Subject name of the certificate."
+            }
+        },
+        "AppServicePlanName": {
+            "type": "string",
+            "metadata": {
+                "description": "App Service Plan Name where certificate will be imported to."
+            }
+        },
+        "Location": {
+            "type": "string",
```

Comment:
add the defaultValue for [resourceGroup().location]

Reply:
done